src/Stmt.cc

```diff
@@ -1290,23 +1290,33 @@ ForStmt::ForStmt(IDPList* arg_loop_vars, ExprPtr loop_expr, IDPtr val_var)
 	{
 	value_var = std::move(val_var);
 
-	if ( e->GetType()->IsTable() )
+	auto t = e->GetType();
+	zeek::TypePtr yield_type;
+
+	if ( t->IsTable() )
+		{
```

Comment:
Formatting nit: Can skip braces around single-line bodies (below too).

Reply:
I force-pushed, I'm not sure it's better, feel free to fixup during the merge.